code/game/dna/mutations/powers.dm

```diff
@@ -0,0 +1,1200 @@
+	activation_messages = list("You notice a strange cold tingle in your fingertips.")
+	deactivation_messages = list("Your fingers feel warmer.")
+	instability = GENE_INSTABILITY_MODERATE
+	spelltype = /obj/effect/proc_holder/spell/targeted/click/cryokinesis
+
+/datum/mutation/grant_spell/cryo/New()
+	..()
+	block = GLOB.cryoblock
+
+/obj/effect/proc_holder/spell/targeted/click/cryokinesis
+	name = "Cryokinesis"
+	desc = "Drops the bodytemperature of another person."
+	panel = "Abilities"
+
+	charge_type = "recharge"
+	charge_max = 1200
+
+	clothes_req = FALSE
+	stat_allowed = FALSE
+
+	click_radius = 0
+	auto_target_single = FALSE	// Give the clueless geneticists a way out and to have them not target themselves
+	selection_activated_message		= "<span class='notice'>Your mind grow cold. Click on a target to cast the spell.</span>"
+	selection_deactivated_message	= "<span class='notice'>Your mind returns to normal.</span>"
+	allowed_type = /mob/living/carbon
+	invocation_type = "none"
+	range = 7
+	selection_type = "range"
+	include_user = TRUE
+	var/list/compatible_mobs = list(/mob/living/carbon/human)
+
+	action_icon_state = "genetic_cryo"
+
+/obj/effect/proc_holder/spell/targeted/click/cryokinesis/cast(list/targets, mob/user = usr)
+
+	var/mob/living/carbon/C = targets[1]
+
+	if(HAS_TRAIT(C, TRAIT_RESISTCOLD))
+		C.visible_message("<span class='warning'>A cloud of fine ice crystals engulfs [C.name], but disappears almost instantly!</span>")
+		return
+	var/handle_suit = FALSE
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(istype(H.head, /obj/item/clothing/head/helmet/space))
+			if(istype(H.wear_suit, /obj/item/clothing/suit/space))
+				handle_suit = TRUE
+				if(H.internal)
+					H.visible_message("<span class='warning'>[user] sprays a cloud of fine ice crystals, engulfing [H]!</span>",
+										"<span class='notice'>[user] sprays a cloud of fine ice crystals over your [H.head]'s visor.</span>")
+				else
+					H.visible_message("<span class='warning'>[user] sprays a cloud of fine ice crystals engulfing, [H]!</span>",
+										"<span class='warning'>[user] sprays a cloud of fine ice crystals cover your [H.head]'s visor and make it into your air vents!.</span>")
+
+					H.bodytemperature = max(0, H.bodytemperature - 100)
+				add_attack_logs(user, C, "Cryokinesis")
+	if(!handle_suit)
+		C.bodytemperature = max(0, C.bodytemperature - 200)
+		C.ExtinguishMob()
+
+		C.visible_message("<span class='warning'>[user] sprays a cloud of fine ice crystals, engulfing [C]!</span>")
+		add_attack_logs(user, C, "Cryokinesis- NO SUIT/INTERNALS")
+
+	//playsound(user.loc, 'bamf.ogg', 50, 0)
+
+	new/obj/effect/self_deleting(C.loc, icon('icons/effects/genetics.dmi', "cryokinesis"))
+
+/obj/effect/self_deleting
+	density = 0
+	opacity = 0
+	anchored = 1
+	icon = null
+	desc = ""
+	//layer = 15
+
+/obj/effect/self_deleting/New(atom/location, icon/I, duration = 20, oname = "something")
+	. = ..()
+	name = oname
+	loc=location
```

Comment:
```suggestion
	loc = location
```

Also dont we already have `/obj/effect/self_deleting` defined?

Reply:
I'll just remove it--we have something similar....also there's just....no sprite, *at all* for this effect.